facade/relayerFacade.go

```diff
@@ -25,6 +27,14 @@ func (rf *relayerFacade) PprofEnabled() bool {
 	return rf.pprofEnabled
 }
 
+func (rf *relayerFacade) GetPeerInfo(pid string) ([]core.QueryP2PPeerInfo, error) {
```

Comment:
missing comments on exported items.
Please add TODOs to implement these functions later.